Configuration/CharacterList.cs

```diff
@@ -9,6 +9,11 @@ namespace Sequencer.Configuration
     [XmlRoot(ElementName = "Words")]
     public class CharacterList : CustomSerializationBaseList<char>
     {
+        public CharacterList()
+        {
+            BackingList = new List<char>();

```

Comment:
Thanks, I missed that. I've removed the redundant constructor and tested to make sure it still works as needed.